src/main/webapp/creatTrip.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Login</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+
+    <body>
+        <h1>New trip</h1>
+        <div class="createTrip">
+            <h4>Start a new trip</h4>
+            <form action="NewTrip" method="POST">
```

Comment:
Ok to leave it as a form for now. You may have to do the post from JS depending on what you want to do after creating the trip. That way you can get the tripId back in the response. You may want to add users to this trip, in which case you will need the tripId.